modules/strelka/1.0/strelka.smk

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env snakemake
+
+
+##### ATTRIBUTION #####
+
+
+# Original Author:  Lauren Chong
+# Module Author:    Helena Winata
+# Contributors:     N/A
+
+
+##### SETUP #####
+
+
+# Import package with useful functions for developing analysis modules
+import oncopipe as op
+
+# Setup module and store module-specific configuration in `CFG`
+# `CFG` is a shortcut to `config["lcr-modules"]["strelka"]`
+CFG = op.setup_module(
+    name = "strelka",
+    version = "1.0",
+    subdirectories = ["inputs", "strelka", "filtered", "outputs"]
+)
+
+# Define rules to be run locally when using a compute cluster
+localrules:
+    _strelka_input_bam,
+    _strelka_input_vcf,
+    _strelka_configure_paired,
+    _strelka_configure_unpaired,
+    _strelka_unzip,
+    _strelka_filter,
+    _strelka_output_vcf,
+    _strelka_all,
+
+
+##### RULES #####
+
+
+# Symlinks the input files into the module results directory (under '00-inputs/')
+rule _strelka_input_bam:
+    input:
+        bam = CFG["inputs"]["sample_bam"],
+        bai = CFG["inputs"]["sample_bai"]
+    output:
+        bam = CFG["dirs"]["inputs"] + "{seq_type}--{genome_build}/{sample_id}.bam",
+        bai = CFG["dirs"]["inputs"] + "{seq_type}--{genome_build}/{sample_id}.bam.bai"
+    run:
+        op.relative_symlink(input.bam, output.bam)
+        op.relative_symlink(input.bai, output.bai)
+
+
+rule _strelka_input_vcf:
+    input:
+        vcf = CFG["inputs"].get("candidate_indel")
```

Comment:
I've leaved against using `.get()` in modules because it will return None rather than triggering an error. 

Actually, does this module currently only run if the candidate indel file is provided? Is there a way to configure it such that it's optional?

Reply:
currently yes, but I can make it optional